packages/components/src/Layout/_Layout.tdc.scss

```diff
@@ -0,0 +1,16 @@
+	@include layout-colors($brand-primary-t7, $brand-secondary-t7);
+
+	[role='banner'] [role='heading'] span {
+		&:before {
+			background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAxNiAxNiIgZmlsbD0iI2ZmZiI+PHBhdGggZD0iTTguODk2IDE0LjYzN2EuOTU3Ljk1NyAwIDAgMC0uNjE0LS41MTJzLS4yMS0uMDY3LS40ODgtLjAwMmEuOTUyLjk1MiAwIDAgMC0uNjIyLjUxNyA2LjY5NSA2LjY5NSAwIDAgMS01Ljc3NS01Ljc4OC45NTMuOTUzIDAgMCAwIC4wMDEtMS42ODggNi42OTQgNi42OTQgMCAwIDEgNS43OS01Ljc4MS45NTIuOTUyIDAgMCAwIDEuNy4wMDMgNi42OTcgNi42OTcgMCAwIDEgNS43NjMgNS43NTUuOTUuOTUgMCAwIDAgLjAwMSAxLjczNCA2LjY5NSA2LjY5NSAwIDAgMS01Ljc1NiA1Ljc2Mm02LjM2My03LjU2QTcuMjk1IDcuMjk1IDAgMCAwIDguOTc1Ljc4My45NTMuOTUzIDAgMCAwIDcuMTAxLjc3OWE3LjI5NCA3LjI5NCAwIDAgMC02LjMwNyA2LjI5Ljk1Ljk1IDAgMCAwLS43OTQuOTM5YzAgLjQ3Mi4zNDMuODYyLjc5My45NGE3LjI5NCA3LjI5NCAwIDAgMCA2LjMxIDYuMjk4Ljk1Mi45NTIgMCAwIDAgMS44NjQtLjAwMiA3LjI5NCA3LjI5NCAwIDAgMCA2LjI5My02LjMwNy45NTQuOTU0IDAgMCAwIDAtMS44NiIvPjxwYXRoIGQ9Ik0xMC45NjYgMTEuMjQ4YTkuMzYgOS4zNiAwIDAgMS0uNjE2LS40NzljLS4zNS0uMjg3LS44My0uNjgtMS4zNzQtLjg4OS0uMzk4LS4xNTMtLjc3OC0uMTg0LTEuMTM0LS4wOTRMOS40MDQgNS43MWMuNzc4LS43NDUgMS41NzItLjIwNiAyLjQ1NC41MThsLjE3Mi4xNDJjLjI1NC4yMDMuNDY1LjM2LjYyNy40NjRsLTEuNjkxIDQuNDE1ek03LjE5NiA5LjU0Yy0uNTczLS44NTktMS43ODItLjkxMi0yLjUyNC0uOTQ0YTExLjM4MyAxMS4zODMgMCAwIDEtLjIxNS0uMDEgOS4xNzUgOS4xNzUgMCAwIDEtLjU2MS0uMDQ2bDEuNjktNC40MTVjLjE5My4wMzEuNDUzLjA1Ni43NzguMDczbC4yLjAxLjAyMy4wMDFjMS4xNC4wNSAyLjA5LjE4IDIuMTcyIDEuMjU0TDcuMTk3IDkuNTR6bTYuMDA1LTMuMTkzYTYuNDg1IDYuNDg1IDAgMCAxLS43MzgtLjUxNmMtLjA0OC0uMDM5LS4xLS4wOC0uMTUxLS4xMjRsLS4wMTYtLjAxM2MtLjYzMi0uNTE5LTEuNzkyLTEuNDczLTIuOTQ3LS44MDUtLjQxNC0xLjI2OS0xLjkxNC0xLjMzNC0yLjczMi0xLjM3bC0uMDQyLS4wMDJhNi41NTUgNi41NTUgMCAwIDEtMS4wNjYtLjExOGMtLjEyLS4wMi0uMTY4LS4wMy0uMjEtLjAyYS4zMzguMzM4IDAgMCAwLS4yNDcuMjEzTDMuMTA3IDguNjY3YS4zNDYuMzQ2IDAgMCAwIC4wODIuMzcyYy4wNjkuMDY2LjE4NC4xNzcgMS4yMy4yMzVsLjIyNC4wMTFjLjc3LjAzMyAxLjgyMi4wOCAyLjA5OS44ODQgMCAwIC4wODQuMjQxLjA4NS4yNGwuNjk1LjI2LjI2LS4xMjNjLjc1Ny0uMzczIDEuNTQuMjczIDIuMTMyLjc2bC4xNy4xNGMuNDMzLjM0Ni43My41NDcuOTA4LjYxNS4wNjcuMDI3LjE2LjAzMi4xNjYuMDMxYS4zNDMuMzQzIDAgMCAwIC4zMDktLjIyMmwxLjk0NS01LjA3NmEuMzQzLjM0MyAwIDAgMC0uMDQxLS4zMjVjLS4wMjUtLjAzNC0uMDY3LS4wNi0uMTctLjEyM3oiLz48cGF0aCBkPSJNMTAuOTcgMTIuMzYyYS40MS40MSAwIDAgMS0uMDUzLS4wMDljLS4wODEtLjAzLS4yOTMtLjE0Ny0uODE0LS41NjlsLS4xNjctLjEzOS0uMTc3LS4xNWMtLjUzOC0uNDY2LTEuMzM4LTEuMTU2LTIuMjk3LS41NDRsLS44MDUtLjMwMkM2LjQ1NSA5LjU5NiA1LjM0MSA5LjU1IDQuNjU4IDkuNTIzYTkuODMzIDkuODMzIDAgMCAxLS4xNzktLjAwOGwtLjIxOS0uMDFjLS44NDQtLjA1My0uOTk3LS4xMzUtMS4wMDgtLjE0MmwtLjAwOC0uMDA4YS4zMzUuMzM1IDAgMSAwLS40NjcuNDhjLjE1NC4xNS4zNTguMjcgMS40NDIuMzM3YTE4OS40NDUgMTg5LjQ0NSAwIDAgMSAuNDEyLjAyYy45NDcuMDM4IDEuMzUuMTQ3IDEuMzg2LjcyMWwuMDE0LjIxNiAxLjUzNy41NzcuMTQ4LS4xMTVjLjU1NS0uNDMyLjk3Ny0uMTM0IDEuNjA2LjQxbC4xODYuMTU4Yy4wNjEuMDUxLjExOS4xLjE3My4xNDMuNDcxLjM4Mi43NzYuNTkuOTkyLjY3NGEuODkzLjg5MyAwIDAgMCAuMjkzLjA1NGguMDFhLjMzNC4zMzQgMCAwIDAtLjAwNi0uNjY4Ii8+PC9zdmc+');
+		}
+	}
+
+	[role='navigation']:before {
```

Comment:
Is that safe to set the image on a role ?
What if in the app, we have other navigation elements ?

Reply:
We can have another `<na>`, but I think we can restrict to one real `<nav role="navigation">` ATM.